apps/web/app/app.tsx

```diff
@@ -52,6 +53,19 @@ export default function App() {
     location => location.pathname === '/' && navigate('/stacking', { replace: true })
   );
 
+  // initAppServices();
+
+  useEffect(() => {
+    import('~/services/init-app-services')
+      .then(({ initAppServices }) => {
+        initAppServices();
+      })
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.error(error);
+      });
+  }, []);
+
```

Comment:
otherwise get error
![Screenshot 2025-05-18 at 11 34 47](https://github.com/user-attachments/assets/838cf61c-48b6-4435-b0da-7ec77b8a2d76)

Reply:
this is strange? why do you know? because buffer has already been polyfilled